docs/guide/windows.md

```diff
@@ -50,3 +50,8 @@ windows_container:
 ```bash
 powershell.exe -EncodedCommand base64(COMMAND)
 ```
+
+## Chocolatey
+
+All containers in Windows Community Cluster come with [Chocolatey](https://chocolatey.org/) preinstalled.
```

Comment:
That's a little bit misleading since `cirrusci/*` windows container like `cirrusci/windowsservercore:2016` have Chocolatey pre-installed. If you'll use your own container without Chocolatey installed on Community Cluster then Chocolatey won't be there. I wonder if we should just mention here that all `cirrusci/*` windows container like `cirrusci/windowsservercore:2016` have Chocolatey pre-installed. 🤔